CSharp/BatchExplorer/Models/ModelBase.cs

```diff
@@ -106,13 +106,17 @@ private List<PropertyModel> ObjectToPropertyModelRecursive(object o, List<string
                         //Don't track null properties in order to keep the list short
                         //propertyModel = new SimplePropertyModel(propInfo.Name, string.Empty);
                     }
+                    else if ((propertyType == typeof(TimeSpan) || propertyType == typeof(TimeSpan?)) && propertyValue.Equals(TimeSpan.MaxValue))
+                    {
+                        propertyModel = new SimplePropertyModel(propInfo.Name, "Unlimited");
```

Comment:
Yeah, it's ok. Just going forward we need to make sure we're consistent because in some places we may show "dev-like" values, and in others we are choosing to make them more non-dev friendly. IMO this was always intended as a dev tool so making things explicit can be beneficial. The dev now needs to reverse engineer Unlimited == Timespan.MaxValue. Not the biggest deal really, but it seems we are preferring one audience over another.
All that said, I'm fine pushing the changes. We can discuss further what the intended audience of this is going forward if we are faced with his kind of decision again.